### ISSUE TITLE
The screen recorded by VLC is reversed horizontally

### ISSUE THREAD

**Issue by marpar98** [rdp/screen-capture-recorder-to-video-windows-free#147]
Hello, I'm using the plug-in screen-capture-recorder on VLC with Directshow, if i select just "open capute device" the result is ok, but actually if I stream it or I save it the video is reversed horizontally. I think the problem could be in the settings on regedit which you can see in the screenshot below on which the resolution setting could be written on the opposite. I report also the low quality of audio of steromix captured. Thank you 
![Screenshot (3)](https://user-images.githubusercontent.com/77637414/104951851-a237e900-59c3-11eb-8557-32430d2d5d03.png)

**Comment by marpar98** [comment]
> Hello, I'm using the plug-in screen-capture-recorder on VLC with Directshow, if i select just "open capute device" the result is ok, but actually if I stream it or I save it the video is reversed horizontally. I think the problem could be in the settings on regedit which you can see in the screenshot below on which the resolution setting could be written on the opposite. I report also the low quality of audio of steromix captured. Thank you

> ![Screenshot (3)](https://user-images.githubusercontent.com/77637414/104951851-a237e900-59c3-11eb-8557-32430d2d5d03.png)

**Comment by rdp** [comment]
Never run into that before.
Does ffmpeg capture it ok?

On Monday, January 18, 2021, marpar98 <notifications@github.com> wrote:
> Hello, I'm using the plug-in screen-capture-recorder on VLC with
Directshow, if i select just "open capute device" the result is ok, but
actually if I stream it or I save it the video is reversed horizontally. I
think the problem could be in the settings on regedit which you can see in
the screenshot below on which the resolution setting could be written on
the opposite. I report also the low quality of audio of steromix captured.
Thank you
> <
https://ci5.googleusercontent.com/proxy/Gnjf8Kdpie-NXZq1hLxStkrUISpAhC24CSJeO8PLNVXatTkFlc-jp9xSwpo7TG-2rTWowUbKa3DxESFm3_ePUDfEtZWop9x6PT6G4eqj0vkwHQHhIjiB-cT1a3CMWh0cyLTDONKqc4sJiZM9dNrkd_07J-4SiX6d=s0-d-e1-ft#https://user-images.githubusercontent.com/77637414/104951851-a237e900-59c3-11eb-8557-32430d2d5d03.png
>
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub, or unsubscribe.<
https://ci3.googleusercontent.com/proxy/Z5doQkjhpPqB6TPEgNrrKnBTNmhq9nFTW-4EkfFSUSjFFFRH1JWr83sjRbDRCDlDN4wPyJrt6PDDcIGamWKDLAAu6z0F2qiQeXP-C5Ed8xkC6vGyv-Eo8qUbwpxUYmpJLjUpb_tVfw-iaycO7QfLa1pkbi9zC-pn__aXkOaejycJKBkd3P-zou4iijK37TOs7keOXrEfZEvEMOFC4qpz1uNJFDw=s0-d-e1-ft#https://github.com/notifications/beacon/AAADBUA245YFSWQ7RQHJYXTS2R46ZA5CNFSM4WHVZTJKYY3PNVWWK3TUL52HS4DFUVEXG43VMWVGG33NNVSW45C7NFSM4LX7FAIA.gif>

**Comment by marpar98** [comment]
> Never run into that before. Does ffmpeg capture it ok?

Yes, screen-capture-recorder doesn't have any problem on capturing, I just didn't understand how to stream it by UDP protocol. The problem comes up when a third part program want to use the video card: VLC (except when I use "open device" function only, OBS studio, Skype...)

**Comment by peci1** [comment]
I see that too with VLC "convert" (which means recording into a file).

**Comment by rdp** [comment]
I have no ideas on this one...

**Comment by romedtino** [comment]
I'm having this issue too and it's driving me nuts lol could it be a 2 monitor issue?

**Comment by rdp** [comment]
Same issue with older vlcs?

On Sunday, February 7, 2021, Jerome Tolentino <notifications@github.com>
wrote:
> I'm having this issue too and it's driving me nuts lol could it be a 2
monitor issue?
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub, or unsubscribe.<
https://ci3.googleusercontent.com/proxy/8kdr4Mg_JA_-zHuI36JmI0cWyWS1PSxzXms4ppNsR2Zg0WP-OvIU-NIu1AgnPV4UJohHTyOREE4J8gmOW32xPwtwmXH1tCx0vN_KkFU3Ss9A-BGAVKwCqaFadJhdgctOWMj7hjCRdGdlJjpvLJtjSNTkyw1mhxrPURogz1VBVHdfSd-zrQ4B9K_Pcv_jO1L_XooPSyG11dMvNCytqLxL-3Pn6LoyuxFQRR6BYBcgSQ=s0-d-e1-ft#https://github.com/notifications/beacon/AAADBUFHIPKJO7O6BQYP7VLS5Y7BVA5CNFSM4WHVZTJKYY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOFYV6DBA.gif>

**Comment by romedtino** [comment]
I'm on the latest version of VLC 3.0.12 and as the OP stated, it seems to only happen when I open a stream or save a video. I tried toying with different start_x, start_y values, switching up the width/height parameters, switching which monitor is my main but no cigar.

**Comment by rdp** [comment]
It works OK with viewing a stream but not OK with saving a video?  Worked OK for me viewing a video just now...
